--- btilde_ggH.f.orig	2014-10-04 17:01:18.015662899 +0200
+++ btilde_ggH.f	2014-10-04 17:01:42.821663624 +0200
@@ -13,6 +13,7 @@
       include 'pwhg_rad.h'
       include 'pwhg_flg.h'
       include 'pwhg_math.h'
+      include 'PhysPars.h'
 c     independent variables for real graph: number of final state
 c     legs times 3, take away 4 for 4-momentum conservation, add 2
 c     for x_1 and x_2, and take away an overall azimuth
@@ -31,7 +32,7 @@
       logical pwhg_isfinite 
       external pwhg_isfinite
       btilde=0
-      www=www0*hc2
+      www=www0*hc2*nnlorescfactor
       do j=1,ndiminteg-3
          xborn(j)=xx(j)
       enddo
